addons/modules/config.cpp

```diff
@@ -0,0 +1,16 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        units[] = {"CBA_ModuleAttack","CBA_ModuleDefend","CBA_ModulePatrol"};
+        requiredVersion = 0.1;
+        requiredAddons[] = {"A3_Modules_F"};
+        author = "$STR_CBA_Author";
+        authors[] = {"WiredTiger"};
+        authorUrl = "";
```

Comment:
Change `authorUrl` to `url = "$STR_CBA_URL";` (compare with how it's done in the other CBA addons)